homeassistant/components/mqtt/server.py

```diff
@@ -85,6 +98,6 @@ def generate_config(hass, passwd):
         username = None
         password = None
 
-    client_config = ('localhost', 1883, username, password, None, PROTOCOL_311)
+    client_config = ('localhost', 1883, username, password, None, '3.1.1')
```

Comment:
"Making more sense" is now making the code more confusing ;-) So to keep doing this you could import the constant inside the generate_config function, however, is the final result that much better?